src/workerd/server/workerd-api.c++

```diff
@@ -446,7 +533,17 @@ void WorkerdApi::compileModules(
     if (hasPythonModules(confModules)) {
       KJ_REQUIRE(featureFlags.getPythonWorkers(),
           "The python_workers compatibility flag is required to use Python.");
-      // Inject pyodide bootstrap module.
+      // Inject Pyodide bundle
+      if(util::Autogate::isEnabled(util::AutogateKey::PYODIDE_LOAD_EXTERNAL)) {
+          if (fetchPyodideBundle(impl->pythonConfig, "dev"_kj)) {
+            modules->addBuiltinBundle(getPyodideBundle("dev"_kj), kj::none);
+          } else {
+            // TODO: hardcoded version number
+            KJ_REQUIRE(fetchPyodideBundle(impl->pythonConfig, "2"_kj), "Failed to get both dev and hardcoded Pyodide version");
+            modules->addBuiltinBundle(getPyodideBundle("2"_kj), kj::none);
+          }
+      }
+      // Inject pyodide bootstrap module (TODO: load this from the capnproto bundle?)
```

Comment:
Do we do so already in EW? If so we should do it here too.

Reply:
I don't think we do. This is the three-line module that does basically nothing